Profiling Python advanced usage warning (#7285)
DataDog/documentation: content/en/tracing/profiling/_index.md

| `DD_PROFILING_TAGS`                              | String        | The tags to apply to an uploaded profile. Must be a list of in the format `<KEY1>:<VALUE1>,<KEY2>:<VALUE2>`.       |
| `DD_SERVICE_NAME`                                | String        | The Datadog [service][3] name, which can be set here, or in `DD_PROFILING_TAGS`.                    |

<div class="alert alert-info">

> Not recommended, advanced usage only.

Reviewer: ```suggestion
Recommended for advanced usage only.
```

</div>

> - If you want to control which part of your code should be profiled, use the `ddtrace.profiling.profiler.Profiler` object:

Reviewer 2: ```suggestion
- If you want to manually control the lifecycle of the profiler, use the `ddtrace.profiling.profiler.Profiler` object:
```

I'd change the phrasing overwise it seems to indicate that if you understand the code you can make sure you profiler only the part you want to profile, which is not what we want customers to do.
